gallery/portfolio.html

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: Portfolio
+section_id: portfolio
+works:
+- image: /images/groupOdd.jpg
+  klass: Team
+- image: /images/homeBanner.jpg
+  klass: Team
+- image: /images/work.png
+  klass: Work
+- image: /images/work2.png
+  klass: Work
+- image: /images/team.jpg
+  klass: Team
+---
+
+
+<div class='full parallax' style='color: #fff;'>
+  <div class='row'>
+    <div class='large-12 columns'>
+      {% include section-header.html title="Our Gallery" class="big" %}
+    </div>
+  </div>
+  <div class='two spacing'></div>
+</div>
+
+
+<div class='full'>
+  <div class='row'>
+    <div class='mod modGallery'>
+      <div class='gallery-nav'>
+        <ul>
+          <li class='current'>
+            <a data-cat='all' href='#'>All</a>
+          </li>
+          <li>
+            <a data-cat='Team' href='#'>Team</a>
+          </li>
+          <li>
+            <a data-cat='Work' href='#'>Work</a>
+          </li>
+        </ul>
+      </div>
+
+      <ul class='gallery medium-block-grid-3'>
+
+        {% for work in page.works %}
+
+        <li style="padding: 25px;" class="{{ work.klass }}">
+          <a href='{{ work.image }}' target="_blank">
+            <img alt="" src="{{ work.image }}" />
+          </a>
+        </li>
+        {% endfor %}
+
+      </ul>
+
+    </div>
+  </div>
+
+  <div class='four spacing'></div>
+</div>
```

Comment:
let's make the images open in the same window as pop ups/with a grey backdrop or something; something like this:
![image](https://user-images.githubusercontent.com/8881141/60443035-57d7f700-9be8-11e9-81f6-3747ce15f664.png)